wiki/Article_styling_criteria/Writing/fr.md

```diff
@@ -282,66 +277,50 @@ Les abréviations, les acronymes et les sigles doivent prendre une majuscule, sa
-
-## Grammar and syntax
+[Les articles avec registres informels](#registre-des-langues) peuvent ignorer cette section ; cependant, il faut éviter de mettre des `Je` dans les paragraphes.
 
-Les articles doivent utiliser de préférence une grammaire et une [syntaxe américaines plus simple](https://www.thepunctuationguide.com/british-versus-american-style.html).
+## Grammaire et syntaxe
 
-### Contractions
+Les articles doivent utiliser de préférence une grammaire et une syntaxe françaises simple.
 
-Les contractions ne doivent pas être utilisées.
+### Nombres
 
-[Les articles avec registres informels](#registre-des-langues) peuvent utiliser des contractions ; toutefois, l'usage doit rester cohérent tout au long de l'article.
-
-### Numbers
-
-*Pour le formatage des nombres, voir [Manuel de style de Wikipédia sur le formatage des nombres](https://en.wikipedia.org/wiki/Wikipedia:Manual_of_Style#Numbers)*
+*Pour le formatage des nombres, voir les [conventions typographiques de Wikipédia concernant les nombres](https://fr.wikipedia.org/wiki/Wikip%C3%A9dia:Conventions_typographiques#Nombres)*
 
 [Les articles avec registres informels](#registre-des-langues) peuvent ignorer cette section ; toutefois, l'usage doit rester cohérent tout au long de l'article.
 
-### Semicolons
+### Points-virgules
 
 [Les articles avec registres informels](#registre-des-langues) doivent éviter d'utiliser les points-virgules (`;`).
 
 ### Virgule en série
 
 La virgule de série, également connue sous le nom de virgule d'Oxford ou de Harvard, doit être utilisée.
 
-### Em dash
-
-Les em dash (`—`) doivent être utilisés sans espace autour d'eux. Les traits d'union ne doivent pas être utilisés à leur place.
-
-### Citations logiques
-
-Consultez le [Manuel de style de Wikipédia pour les citations logiques](https://en.wikipedia.org/wiki/Wikipedia:Manual_of_Style#Punctuation_inside_or_outside).
+### Citations
 
-Extrait du manuel de style de Wikipédia :
+Consultez les [conventions typographiques de Wikipédia concernant les citations](https://fr.wikipedia.org/wiki/Wikip%C3%A9dia:Citation).
 
-> N'incluez la ponctuation terminale à l'intérieur des guillemets que si elle était présente dans le texte original, sinon placez-la après le guillemet fermant. Dans la plupart des cas, cela signifie qu'il faut traiter les points et les virgules de la même manière que les points d'interrogation : les placer à l'intérieur des guillemets s'ils ne s'appliquent qu'au matériel cité et à l'extérieur s'ils s'appliquent à l'ensemble de la phrase.
+N'incluez la ponctuation à l'intérieur des guillemets que si elle était présente dans le texte original, placez-la après le guillemet fermant dans le cas contraire. Dans la plupart des cas, cela signifie qu'il faut traiter les points et les virgules de la même manière que les points d'interrogation : les placer à l'intérieur des guillemets s'ils ne s'appliquent qu'au matériel cité et à l'extérieur s'ils s'appliquent à l'ensemble de la phrase.
```

Comment:
if this part isn't in the french wikipedia article, mention that it is translated from the english one above and leave the angle bracket for the blockquote